spec/unit/verifier_spec.rb

```diff
@@ -75,6 +75,24 @@
         }.to output(expected_output).to_stdout
       end
     end
+
+    context "missing topics" do
```

Comment:
This is the test for the abstract function of adding warnings. It is not limited to warnings about topic. So the context and the names of the test should be consistent with the other tests. The context would rather be something like "one warning" and the test something like "prints warning".